taxi/forms.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ["license_number", ]
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),

```

Comment:
The `queryset` for the `drivers` field in `CarForm` is set to `get_user_model().objects.all()`. This might not be correct if you want to filter only `Driver` instances. Consider using `Driver.objects.all()` instead to ensure only drivers are selectable.